src/platform/backends/lxd/lxd_virtual_machine_factory.h

```diff
@@ -63,4 +63,9 @@ class LXDVirtualMachineFactory : public BaseVirtualMachineFactory
 };
 } // namespace multipass
 
+inline void multipass::LXDVirtualMachineFactory::require_suspend_support() const
```

Comment:
You should be able to write a unit test that catches that an exception is thrown here.